features/frontend/checkout_security.feature

```diff
@@ -96,3 +96,13 @@ Feature: Checkout security
           And I press "Register"
          Then I should be on the checkout security forward step
           And I should see "Please enter your email"
+
+    Scenario: Placing order as guest
```

Comment:
I would say we need one more scenario, placing order as guest for already existing customer or is it covered somewhere else?